CODEOWNERS

```diff
@@ -13,11 +13,61 @@ go.mod @fleetdm/go
 # GitHub settings + actions
 /.github/ @fleetdm/g-platform
 
+# Codeowners file
+/CODEOWNERS @zwass @mikermcneil
+
 # Changelog
-CHANGELOG.md @noahtalerman
+/CHANGELOG.md @noahtalerman
 
-# Markdown docs
-*.md @Desmi-Dizney
+# Fleet documentation
+/docs/ @Desmi-Dizney @chris-mcgillicuddy
```

Comment:
I intentionally made Desmi a codeowner for all markdown files so that even documentation outside of the main docs (eg. READMEs specific to directories of code, handbook, etc.) will see his review.

Reply:
Context: Des isn't added as a reviewer prior to merge for other markdown files- he reviews them post facto, after they have been merged.

Des is only the reviewer for PRs to docs so that he can route to the oncall engineer.  (That's to save the submitter the time and friction of figuring out who the oncall engineer is.  And in some cases, Des will be able to merge right away.)